src/__stories__/select-story.tsx

```diff
@@ -15,6 +15,7 @@ type Args = {
     error: boolean;
     native: boolean;
     optional: boolean;
+    showOptionalLabel: boolean;
```

Comment:
mmm why not hideOptionalLabel? the default state is show the label when you use optional={true}, and using this prop you want to hide it.

However, if you use `<TextField name="name" label="Name" showOptionalLabel={true} />` the label is not shown (it is obviously) but I hope that the prop will show the label with your approach.

Reply:
If its required it shouldn't show the label no matter the prop value is IMO so current behaviour is expected.

Reply:
I think I prefer positive booleans, even if the default value is true.

Of course, if the field isn't optional the label is never shown, but when it is the label is shown unless you set `showOptionalLabel` to false